Library/Homebrew/test/utils/inreplace_spec.rb

```diff
@@ -7,7 +7,7 @@
   let(:file) { Tempfile.new("test") }
 
   before do
-    file.write <<~EOS
```

Comment:
I think this has always been broken, but the existing tests in the file only tested error conditions. Note that `inreplace` uses [binread](https://github.com/Homebrew/brew/blob/fa384b7/Library/Homebrew/utils/inreplace.rb#L56), so we should use the corresponding write operation here.